files.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+class FileAnalyzer
+  def initialize(path)
+    @file = path
+    count_empty_lines
```

Comment:
Good idea, I'll implement this in my code.

Reply:
left dividing File into lines in methods, because it was causing such output:
Empty lines: 0
Lines count: 5
File has 5 y
Empty lines: 1
Lines count: 5
File has 5 y
Empty lines: 0
Lines count: 5
File has 5 y
Empty lines: 0
Lines count: 5
File has 5 y
Empty lines: 0
Lines count: 5
File has 5 y

Reply:
It is just an example, sure we need to rewrite it, it wouldn't be interesting if I give you ready-made solutions :)

Reply:
Hi, I created one method which gives us all statistics. Please, review when you have time)

Reply:
Super! I like it! 
What will be the output if I entered an invalid file name etc?

Reply:
Thank you, for reminding about that! 
Added message in case file is invalid.